app/views/proposals/details/_top_notification.html.haml

```diff
@@ -0,0 +1,6 @@
+.medium-12.top-notification-wrapper
+  .row
+    .medium-12.column
+      = image_tag 'icon-configuration.png'
+      %p
+        This is a beta version of the Request Details page. #{link_to 'Switch to previous design', proposals_revert_design_path} if you prefer.
```

Comment:
@pkarman I didnt explore that idea originally. I just coded up what that would look like.

Do you think it would be better to do via JavaScript? My only concern would be that the cookie is set in the controller from the first place. I see the single-use route being added as a 'con', but not a show stopper.

Thoughts?